icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -39,7 +39,7 @@ private static int getIndex(String pluralKeyword) {
         }
     }
 
-    private static String getWithPlural(String[] strings, StandardPlural plural) {
+    protected static String getWithPlural(String[] strings, StandardPlural plural) {
```

Comment:
This function is "protected" rather than "private" because MixedUnitLongNameHandler is referencing it. It apparently doesn't need to be public.